examples/clients/clients.py

```diff
@@ -19,10 +20,10 @@
 # Retrieve stats by a specific client type. #
 # GET /clients/{client_type}/stats #
 
-params = {'aggregated_by': 'day', 'start_date': '2016-01-01', 'end_date': '2016-04-01'}
+params = {'aggregated_by': 'day',
```

Comment:
Would it be ok to be pedantic here and ask that end_date be moved to the next line too?

Reply:
Updated.